Cms/ProfileBioEdit.ascx.cs

```diff
@@ -0,0 +1,2419 @@
+            {
+                var rockContext = new RockContext();
+
+                if ( !_groupId.HasValue )
+                {
+                    // GroupId wasn't specified due to invalid situation
+                    // Return and report nothing.
+                    return;
+                }
+
+                // reload group in new Context to be able to update values
+                var group = new GroupService( rockContext ).Get( _groupId.Value );
+                if ( group == null )
+                {
+                    return;
+                }
+
+                var avcFamilyAttributes = pnlProfilePanels.FindControl( "avcFamilyAttributes" ) as AttributeValuesContainer;
+
+                var wrapTransactionResult = rockContext.WrapTransactionIf( () =>
+                {
+                    Person person = CurrentPerson;
+
+                    var returnVal = SavePerson( rockContext, ref person, pnlProfilePanels, group );
+
+                    if ( returnVal )
+                    {
+                        var pnlFamilyMemberBody = pnlProfilePanels.FindControl( "pnlFamilyMemberBody" );
+                        if ( pnlFamilyMemberBody != null )
+                        {
+                            foreach ( var fm in FamilyMembers )
+                            {
+                                var pwFamilyMember = pnlFamilyMemberBody.FindControl( $"pwFamilyMember_{fm.PersonId}" ) as PanelWidget;
+                                var fmPerson = fm.Person;
+                                if ( pwFamilyMember != null )
+                                {
+                                    var pnlFamilyMemberPerson = pwFamilyMember.FindControl( "pnlFamilyMemberPerson" ) as Panel;
+                                    returnVal = SavePerson( rockContext, ref fmPerson, pnlFamilyMemberPerson, group );
+                                    if ( returnVal )
+                                    {
+                                        fm.Person = fmPerson;
+                                        pwFamilyMember.Expanded = false;
+                                        pwFamilyMember.ShowDeleteButton = false;
+                                        var rblRole = pnlFamilyMemberPerson.FindControl( "rblRole" ) as RockRadioButtonList;
+                                        var roleText = fm.Person.AgeClassification != AgeClassification.Unknown ? fm.Person.GetFamilyRole().ToString() : rblRole?.SelectedItem?.Text;
+                                        pwFamilyMember.Title = $"<h6>{fm.Person.FullName}<br><small>{roleText}</small></h6>";
+                                    }
+                                    else
+                                    {
+                                        break;
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    if ( group != null )
```

Comment:
Test person in multiple family groups.

Reply:
@guymaness2 tested, works as it should. Ready to merge!